### PR TITLE
ACM-5054  Add support for baremetal clusters

### PR DESCRIPTION
**Pull request by aswinsuryan**
Add support for baremetal clusters

### CODE REVIEW

File: frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+import {useContext, useEffect, useState} from 'react'
+import {useHistory} from 'react-router'
+import {Link} from 'react-router-dom'
+import {AcmDataFormPage} from '../../../../../../components/AcmDataForm'
+import {FormData, Section} from '../../../../../../components/AcmFormData'
+import {RbacButton} from '../../../../../../components/Rbac'
+import {Trans, useTranslation} from '../../../../../../lib/acm-i18next'
+import {DOC_LINKS} from '../../../../../../lib/doc-util'
+import {rbacCreate} from '../../../../../../lib/rbac-util'
+import {validateCidr, validateCloudsYaml, validateJSON} from '../../../../../../lib/validation'
```
Comment (yboaron) [inline frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx:14]: Do we need this change ?

Comment (aswinsuryan) [reply frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx]: It was done by IDE during save, removed it.

**Review by yboaron — APPROVED** [review]

**Pull request merged**